Godeps

```diff
@@ -2,3 +2,5 @@ github.com/nsf/termbox-go bc836fdbcf75184c95c96500ea740e3dd727b5a8
 github.com/boltdb/bolt da700e8fdb065b7e073700c9afa8369d4c60c674
```

Comment:
@dw-vsn Why is bolt still in here? I thought we decided against it for Canary? Or did you have it in mind for something else?